docs/configuration.md

```diff
@@ -0,0 +1,87 @@
+<!--
+    Copyright 2019 VMware, Inc.
+    SPDX-License-Identifier: BSD-2-Clause
+-->
+
+# Precaution configuration
+
+Precaution uses a variety of tools under the hood without exposing
+implementation details and complexity of the underlying tools to the user.
+
+Concretely, this means that Precaution ignores any tool specific configuration
```

Comment:
Why "concretely"?

Reply:
Because of this sentence is connected with the statement above.
“Precaution uses a variety of linters under the hood without exposing
implementation details and complexity of the underlying tools to the user **_and because of that_** we ignore linter specific configuration files”.

Reply:
I like **and because of that** better than having a separate statement starting with "concretely".

I think even better would be to expand on the reasoning a bit.

> Precaution uses a variety of linters under the hood without exposing implementation details and complexity of the underlying tools to the user. Because of that, and in order to prevent conflict with any configuration files which may exist in the target git repository, Precaution ignores any linter specific configuration files.